src/common/options/global.yaml.in

```diff
@@ -6092,4 +6092,4 @@ options:
   services:
   - rgw
   - osd
-  with_legacy: true
\ No newline at end of file
+  with_legacy: true
```

Comment:
this change was unintentional?

Reply:
Unintentional but it just add '\n' to the last line and I considered it as harmless. I'll remove this change if necessary.

Reply:
please make this change in a separate commit because it does not need to be backported